app.py

```
@@ -72,22 +72,26 @@ def template():
-    # result, result_class = one_page(title, wiki)
+    if not title:
+        return render_template("result.html", title=title, url=url, result_class="danger", tt="title_required"), 400
 
     try:
-        result1, result_class = one_page(title, wiki)
-        return render_template("result.html", title=title, url=url, result_class=result_class, tt=result1, tt1=""), 200
+        result1 = one_page(title, wiki)
+        # ---
+        result_class = result1.get("result_class", "")
+        result_text = result1.get("result_text", "")
+        length = result1.get("length", 0)
```

Ruff (0.8.2)

Local variable `length` is assigned to but never used

Remove assignment to unused variable `length`

(F841)